veracruz-server/src/lib.rs

```diff
@@ -30,3 +30,10 @@ pub mod veracruz_server_nitro;
 #[cfg(feature = "nitro")]
 pub use self::veracruz_server_nitro::veracruz_server_nitro::*;
 
+#[cfg(feature = "linux")]
+pub mod veracruz_server_linux;
+#[cfg(feature = "linux")]
+pub use self::veracruz_server_linux::veracruz_server_linux::*;
```

Comment:
Yes, I dislike this pattern too (though I think in this case the `*` is being used to re-export the content of the `veracruz_server_linux` module, here).  Another pattern I dislike is using qualified names in the body of a module (e.g. writing `let foo: std::vec::Vec<_> = std::vec::Vec::new();` instead of hoisting the import to the top of the file and guarding it with a feature branch, as appropriate.  I think the latter pattern is used more, though, than the former and may be a bigger challenge to change, now.